chat-plugins/modlog.js

```diff
@@ -304,8 +317,10 @@ exports.commands = {
 			searchString = searchString.substring(1, searchString.length - 1);
 		}
 
-		PM.send(roomId, searchString, exactSearch, lines).then(response => connection.popup(prettifyResults(response, roomId, searchString, exactSearch, addModlogLinks, hideIps)));
-		if (cmd === 'timedmodlog') this.sendReply(`The modlog query took ${Date.now() - startTime} ms to complete.`);
+		PM.send(roomId, searchString, exactSearch, lines).then(response => connection.popup(prettifyResults(response, roomId, searchString, exactSearch, addModlogLinks, hideIps)))
+			.then(() => {
+				if (cmd === 'timedmodlog') this.sendReply(`The modlog query took ${Date.now() - startTime} ms to complete.`);
+			});
```

Comment:
I mean, technically this does what you want it to do, but it's like writing `if (a == true)` instead of `if (a)`